source/_docs/installation/virtualenv.markdown

```diff
@@ -1,20 +1,27 @@
 ---
 layout: page
-title: "Installation in virtualenv"
-description: "Instructions how to install Home Assistant in a virtual environment."
+title: "Installation: Python virtual envenvironment"
+description: "How to install Home Assistant in a virtual environment."
 date: 2016-4-16 16:40
 sidebar: true
 comments: false
 sharing: true
 footer: true
 redirect_from: /getting-started/installation-virtualenv/
 ---
+<p class='note'>
+Note: Beginners should check our [Getting started guide](/getting-started/) first. This is for users that require a flexible installation or want to run Home Assistant on any platform.
```

Comment:
Remove `Note:`. Again, `platform`...no no differentiation about hardware or software. Better leave it out. The note addresses the beginner and shouldn't confuse them.